src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -62,10 +74,12 @@ export class ListingDataTableComponent extends BaseComponent {
      * @memberof ListingDataTableComponent
      */
     loadData(limit: number, offset: number, sortField?: string, sortOrder?: number, query?: string): void {
-        this.crudService.loadData(this.url, limit, offset, sortField,
-        sortOrder < 0 ? OrderDirection.DESC : OrderDirection.ASC, query)
-            .subscribe( response => {
-                this.items = response.items;
+        if (sortField) {
+            sortField = this.defaultSortOrder;
```

Comment:
This is not right, it should be:
if (!sortField) {
    sortField = this. defaultSortField;
}

or ...

sortField = !sortField ? this. defaultSortField : sortField;